src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -2145,12 +2165,33 @@ private ushort getRelocTypeHint(IntPtr _this, void* target)
             return UInt16.MaxValue;
         }
 
-        private void getModuleNativeEntryPointRange(IntPtr _this, ref void* pStart, ref void* pEnd)
+        private void getModuleNativeEntryPointRange(ref void* pStart, ref void* pEnd)
         { throw new NotImplementedException("getModuleNativeEntryPointRange"); }
 
-        private uint getExpectedTargetArchitecture(IntPtr _this)
+        private uint getExpectedTargetArchitecture()
         {
             return 0x8664; // AMD64
         }
+
+        [UnmanagedFunctionPointerAttribute(CallingConvention.ThisCall)]
+        delegate void _getLocationOfThisType_wrapper_Windows(IntPtr _this, CORINFO_LOOKUP_KIND* result, out IntPtr exception, CORINFO_METHOD_STRUCT_* context);
+
+        void FixupUnmanagedInstance()
+        {
+            if (IsWindows())
+            {
+                IntPtr * vtable = *(IntPtr**)_comp;
+                for (int i = 0; i < _keepalive.Length; i++)
+                {
```

Comment:
Added the ability to manually write C++ wrappers. Auto-generated wrappers are in jitinterface.h, while manually written ones are in jitinterface.cpp.